doc/stdenv/stdenv.chapter.md

```diff
@@ -116,6 +116,65 @@ On Linux, `stdenv` also includes the `patchelf` utility.
 
 ## Specifying dependencies {#ssec-stdenv-dependencies}
 
+Build systems often require more dependencies than just what `stdenv` provides. This section describes attributes accepted by `stdenv.mkDerivation` that can be used to make these dependencies available to the build system.
+
+### Simplified overview {#ssec-stdenv-dependencies-overview}
+A full reference of the many kinds of dependencies needed to cover all possible cases is provided in the next section, but here is a simplified overview of the most common kinds. It should cover most derivations.
```

Comment:
```suggestion
A full reference of the different kinds of dependencies is provided in the next section, but here is an overview of the most common ones.
It should cover most use cases.
```

We don't have to make it sound overly complex, even if it is, and should use a neutral tone in the reference manual. We're just stating facts.